spec/models/news_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe News, type: :model do
+  let!(:news) { create(:news) }
```

Comment:
Maybe the `let!` and `create` is not necessary in this kind of tests.
```suggestion
  let!(:news) { create(:news) }
  let(:news) { build(:news) } 
```